aries-site/src/examples/components/layer/LayerSideDrawerExample.js

```diff
@@ -114,16 +121,32 @@ export const LayerSideDrawerExample = () => {
   const size = useContext(ResponsiveContext);
   const onOpen = () => setOpen(true);
   const onClose = () => setOpen(undefined);
+
   return (
     <>
       <Box align="start">
-        <Button label="Show me the side drawer" onClick={onOpen} primary />
+        <AnnounceContext.Consumer>
+          {announce => (
+            <Button
+              label="Show me the side drawer"
+              onClick={() => {
+                onOpen();
+                //             announce(
+                //               `You are in a layer containing a form titled
+                // Form Header, a subtitle if needed.`,
+                //               'assertive',
+                //             );
+              }}
+              primary
+            />
+          )}
+        </AnnounceContext.Consumer>
```

Comment:
Should this be reverted since you are taking an alternate approach?

Reply:
Thank you let me cleanup.